### DISCUSSION TITLE
Disable landing page?

### DISCUSSION THREAD

**Discussion by flamingm0e** [colinwilson/lotusdocs#137] [Q&A]
Spinning up a documentation site for my home lab and it's been several years since I played with Hugo.

Is there a way to just disable the landing page?  Make the "root" be my content?  I feel like this is a setting in Hugo that I just cannot seem to locate.  I love lotusdocs, but I don't need a landing page for this project.

**Comment by dmtrkzntsv** [comment]
You can configure a redirect to the `/docs/` URL as a temporary/permanent solution

**Comment by fenglisch** [comment]
See https://github.com/colinwilson/lotusdocs/issues/170
